packages/rpc-provider/src/http/index.ts

```diff
@@ -56,6 +56,13 @@ export default class HttpProvider implements ProviderInterface {
     return false;
   }
 
+  /**
+   * @description Manually disconnect from the connection
+   */
+  disconnect (): void {
+    throw new Error('Unimplemented');
```

Comment:
I think a nop is fine in the sense there are no ongoing connections to disconnect?
Unless this suppose to disconnect any outgoing requests that have not responded yet?